### DISCUSSION TITLE
[HW 4] About sst2 dataset

### DISCUSSION THREAD

**Discussion by sklee2014** [seominjoon/kaist-ai605#59] [Q&A] [answered]
In order to solve problem 2, I've loaded SST-2 dataset as given in the example.

If sst2_dataset = load_dataset('glue', 'sst2'), then sst2_dataset was some kind of dictionary with keys ['train', 'validation', 'test'].

There was no problem in 'train' and 'validation' (sst2_dataset['train']['label'] or sst2_dataset['validation']['label']), but when I checked 'test' (sst2_dataset['test']['label']) all the labels were -1.

Would there be some problem in the SST-2 dataset? Or should I use 'validation' for reporting the accuracy?

**Comment by seominjoon** [comment, marked as answer]
I apologize for the confusion. This is because GLUE doesn't want you to look into the test data (which is used for the leaderboard). But also note that the original test data is public, though in a slightly different format from GLUE. For instance, you can access the test data with `load_dataset('sst')`. 

But rather than trying to use this test data, please consider the `validation` data as the test data. It might not be an apple-to-apple comparison with Assignment 1, but the difference will be easily noticeable so it is okay.
